src/main/java/mate/academy/Main.java

```diff
@@ -39,17 +50,48 @@ public static void main(String[] args) {
         tomorrowMovieSession.setMovie(fastAndFurious);
         tomorrowMovieSession.setShowTime(LocalDateTime.now().plusDays(1L));
 
+        MovieSession plusTwoDaysMovieSession = new MovieSession();
+        plusTwoDaysMovieSession.setCinemaHall(secondCinemaHall);
+        plusTwoDaysMovieSession.setMovie(fastAndFurious);
+        plusTwoDaysMovieSession.setShowTime(LocalDateTime.now().plusDays(2L));
+
         MovieSession yesterdayMovieSession = new MovieSession();
         yesterdayMovieSession.setCinemaHall(firstCinemaHall);
         yesterdayMovieSession.setMovie(fastAndFurious);
         yesterdayMovieSession.setShowTime(LocalDateTime.now().minusDays(1L));
 
-        MovieSessionService movieSessionService = null;
+        MovieSessionService movieSessionService =
+                (MovieSessionService) injector.getInstance(MovieSessionService.class);
         movieSessionService.add(tomorrowMovieSession);
         movieSessionService.add(yesterdayMovieSession);
+        movieSessionService.add(plusTwoDaysMovieSession);
```

Comment:
Adding `yesterdayMovieSession` to the `movieSessionService` might not be intended since it represents a past session. Consider whether this is necessary or if it should be excluded.